scripts/zones/Chateau_dOraguille/npcs/Halver.lua

```diff
@@ -102,7 +102,7 @@ function onTrigger(player,npc)
 		-- Mission San D'Oria 2-3 Journey Abroad 
 		elseif(currentMission == JOURNEY_ABROAD and MissionStatus == 11) then
 			player:startEvent(0x01fb);
-		elseif(currentMission == JOURNEY_ABROAD and MissionStatus == 0) then
+		elseif(currentMission == JOURNEY_ABROAD and MissionStatus == 1) then

```

Comment:
If someone was mid mission when he got changed from 1 to 0 he will be broken for that player. Likewise, this change breaks him for the next batch of people.
https://github.com/DarkstarProject/darkstar/pull/1247
He's been changed back and forth multiple times from ppl "fixing" him at this exact spot.